### PR TITLE
add a responsive header with logo and title

### PR DESCRIPTION
**Pull request by pricealexandra**
also start a theme and breakpoints and add some global styling including font and text color

### CODE REVIEW

File: components/header.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import Image from 'next/image';
+import logo from '../public/aclima-logo.svg';
+import theme from '../utils/theme';
+
+export const Header: React.FC = () => {
+  return (
+    <div>
+      <Image src={logo} alt="Aclima logo" />
+      <h1>Example Prototype</h1>
+      <style jsx>{`
```
Comment (pricealexandra) [inline components/header.tsx:10]: I don't mind styled jsx! It's not quite as compact or pretty as styled components but I feel like it's a good compromise between plain old css and more developer friendly options.

File: utils/theme.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { Source_Sans_3 } from 'next/font/google';
+import { text } from 'stream/consumers';
+
+export const SourceSans = Source_Sans_3({
+  weight: ['400', '600', '700'],
+  subsets: ['latin'],
+  variable: '--font-source-sans',
+});
+
+const prototypeStylingTheme = {
```
Comment (pricealexandra) [inline utils/theme.tsx:10]: no doubt Joshua will want to go to town on this...for now I've arbitrarily cherry-picked some colors from the Pro theme.

**Comment by pricealexandra** [comment]
will do a sidebar next!

File: components/header.tsx (hunk 1 above)
Comment (paigewilliams) [reply components/header.tsx:10]: I'm excited to try it out!

File: utils/theme.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import { Source_Sans_3 } from 'next/font/google';
```
Comment (paigewilliams) [inline utils/theme.tsx:1]: woo

File: pages/index.tsx [hunk 4]
```diff
@@ -16,7 +18,20 @@ export default function Home() {
         <meta name="viewport" content="width=device-width, initial-scale=1" />
         <link rel="icon" href="/favicon.ico" />
       </Head>
-      <main style={{ width: '100%', height: '100vh' }}>
+      <main className={SourceSans.className}>
```
Comment (paigewilliams) [inline pages/index.tsx:21]: cool!

**Review by paigewilliams — APPROVED** [review]
💅🏻

**Pull request merged**